Sources/Hero.swift

```diff
@@ -391,6 +391,16 @@ extension Hero: UINavigationControllerDelegate {
   public func navigationController(_ navigationController: UINavigationController, interactionControllerFor animationController: UIViewControllerAnimatedTransitioning) -> UIViewControllerInteractiveTransitioning? {
     return interactiveTransitioning
   }
+  
+  public func navigationController(_ navigationController: UINavigationController, willShow viewController: UIViewController, animated: Bool) {
+    // Forward to previousNavigationDelegate
+    navigationController.previousNavigationDelegate?.navigationController?(navigationController, willShow: viewController, animated: animated)
```

Comment:
This might not be the behavior that everyone wants. Can we have a boolean variable that enables this behavior.